source/Mocka/Classes/ClassMock.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace Mocka\Classes;
+
+use CodeGenerator\ClassBlock;
+use Mocka\Exception;
+use Mocka\Invokables\Invokable\Stub;
+use Mocka\Overrides\Manager;
+use Mocka\Overrides\MethodOverrides\ClassOverrides;
+
+class ClassMock {
+
+    /** @var string */
+    private $_name;
+
+    /** @var string|null */
+    private $_namespace;
+
+    /** @var string|null */
+    private $_parentClassName;
+
+    /**
+     * ClassMock constructor.
+     * @param string      $wrapperClassName
+     * @param string|null $className
+     */
+    public function __construct($wrapperClassName, $className = null) {
```

Comment:
`wrapperClassName` is a bit misleading. At first I thought it was the custom name that you can give to a classMock (because it wraps the mocked class, not the other way around?), until I looked at the constructor.